web/frontend/src/components/base.vue

```diff
@@ -82,7 +82,6 @@ const stop = () => {
   stopped = true;
   const req = new baseApi.StopRequest();
   req.setName(props.name);
-  rcLogConditionally(req);
```

Comment:
we should leave this in, Steve uses 'em often

Reply:
fixed